nautobot_floor_plan/forms.py

```diff
@@ -27,6 +27,17 @@ class FloorPlanForm(NautobotModelForm):
 
     location = DynamicModelChoiceField(queryset=Location.objects.all())
 
+    x_origin_seed = forms.CharField(
+        label = "Beginning X value",
```

Comment:
```suggestion
        label = "X Axis Seed",
```

Making this consistent with the detail view.